lib/urlwatch/reporters.py

```diff
@@ -383,6 +383,8 @@ class EMailReporter(TextReporter):
 
     def submit(self):
         filtered_job_states = list(self.report.get_filtered_job_states(self.job_states))
+        cfg = self.report.config['report']['email']
+        subject_length = cfg['subject_length']
```

Comment:
This needs to deal with situations where `subject_length` is not set (e.g. old configuration where this key didn't exist). Also, it's probably useful to make it so that there is an option of setting the value to `null` or something so that the subject length isn't limited.